config.yml.default

```diff
@@ -1,131 +1,131 @@
+      move_quality: "best"         # One of "good", "best", "suggest" (it takes all the "good" moves and tells the engine to only consider these; will move instantly if there is only 1 "good" move).
 
-  lichess_bot_tbs:           # The tablebases list here will be read by lichess-bot, not the engine.
+  lichess_bot_tbs:                 # The tablebases list here will be read by lichess-bot, not the engine.
     syzygy:
       enabled: false
-      paths:
+      paths:                       # Paths to Syzygy endgame tablebases.
         - "engines/syzygy"
-      max_pieces: 7
-      move_quality: "best"   # One of "good", "best", "suggest" (it takes all the "good" moves and tells the engine to only consider these; will move instantly if there is only 1 "good" move).
+      max_pieces: 5                # Maximum number of pieces in the endgame tablebase.
+      move_quality: "best"         # One of "good", "best", "suggest" (it takes all the "good" moves and tells the engine to only consider these; will move instantly if there is only 1 "good" move).
     gaviota:
       enabled: false
       paths:
         - "engines/gaviota"
       max_pieces: 5
       min_dtm_to_consider_as_wdl_1: 120  # The minimum dtm to consider as syzygy wdl=1/-1. Set to 100 to disable.
-      move_quality: "best"   # One of "good", "best", "suggest" (it takes all the "good" moves and tells the engine to only consider these; will move instantly if there is only 1 "good" move).
+      move_quality: "best"         # One of "good", "best", "suggest" (it takes all the "good" moves and tells the engine to only consider these; will move instantly if there is only 1 "good" move).
 
-# engine_options:            # Any custom command line params to pass to the engine.
+# engine_options:                  # Any custom command line params to pass to the engine.
 #   cpuct: 3.1
 
   homemade_options:
 #   Hash: 256
 
-  uci_options:               # Arbitrary UCI options passed to the engine.
-    Move Overhead: 100       # Increase if your bot flags games too often.
-    Threads: 2               # Max CPU threads the engine can use.
-    Hash: 256                # Max memory (in megabytes) the engine can allocate.
-    SyzygyPath: "./syzygy/"  # Paths to Syzygy endgame tablebases
-#   go_commands:             # Additional options to pass to the UCI go command.
-#     nodes: 1               # Search so many nodes only.
-#     depth: 5               # Search depth ply only.
-#     movetime: 1000         # Integer. Search exactly movetime milliseconds.
-
-# xboard_options:            # Arbitrary XBoard options passed to the engine.
+  uci_options:                     # Arbitrary UCI options passed to the engine.
+    Move Overhead: 500             # Increase if your bot flags games too often.
+    Threads: 4                     # Max CPU threads the engine can use.
+    Hash: 512                      # Max memory (in megabytes) the engine can allocate.
```

Comment:
I don't think the engine options should be changed.

Reply:
Most PCs these days have 8 threads or more, and atleast 4GB RAM. Even a Raspberry Pi meets those requirements, that is why I have changed. Hopefully it is fine now.

Reply:
You can leave `Threads` and `Hash` as you have them, but `Move Overhead` set to 500 is too much. You decreased `move_overhead` to prevent blunders, but increasing `Move Overhead` will lead to more blunders.